build/yaml/deployBotResources/dotnet/composerPrepare.yml

```diff
@@ -0,0 +1,36 @@
+parameters:
+  project: {}
+  output: {}
+  
+steps:
+# Prepare DotNet deploy
+- task: AzureCLI@1
+  displayName: 'Prepare Deploy'
+  inputs:
+    azureSubscription: $(AzureSubscription)
+    scriptLocation: inlineScript
+    inlineScript: 'az bot prepare-deploy --lang Csharp --code-dir ${{ parameters.project.directory }}/runtime/azurewebapp --proj-file-path Microsoft.BotFramework.Composer.WebApp.csproj --output json'
+
+# Build Bot
+- task: DotNetCoreCLI@2
+  displayName: 'Build'
+  inputs:
+    command: publish
+    publishWebProjects: false
+    projects: '${{ parameters.project.directory }}/runtime/azurewebapp/Microsoft.BotFramework.Composer.WebApp.csproj'
+    arguments: '-c release -o ${{ parameters.output.path }}'
+    modifyOutputPath: false
+    zipAfterPublish: false
+
+# Arrange composer folders & files
+- powershell: |
+    $composerDialogs = "${{ parameters.output.path }}/ComposerDialogs"
+    Remove-Item $composerDialogs -Recurse -ErrorAction Ignore
+
+    Copy-Item -Path (Get-Item -Path ${{ parameters.project.directory }}/* -Exclude ('runtime', 'generated')).FullName -Destination $composerDialogs -Recurse -Force -Container
```

Comment:
Let's not exclude generated (not sure yet but I think we may need it).
Let's exclude the scripts folder (that's only used by the composer app for local deployment)
There are a couple of files that could be ignored too: 
* *.ps1
* readme.md
* *.botproj
* .gitignore

Reply:
Hi @gabog, we updated the script to exclude these files and keep the 'generated' folder. Thanks!